RELEASING.rst

```diff
@@ -100,16 +100,17 @@ Releasing
 
 Both automatic and manual processes described above follow the same steps from this point onward.
 
-#. After all tests pass and the PR has been approved, tag the release commit
-   in the ``MAJOR.MINOR.x`` branch and push it. This will publish to PyPI::
+#. After all tests pass and the PR has been approved, merge the PR.
 
+#. Tag the merge commit in the ``MAJOR.MINOR.x`` branch and push it. This will publish to PyPI::
```

Comment:
Here I say to tag the merge commit but can also tag the release commit itself. The merge commit seems more appropriate to me, but I see previous releases tagged the release commit. Going to keep it as the merge commit but can change it.